### ISSUE TITLE
Filters on device around me

### ISSUE THREAD

**Issue by nihilistau** [Semper-Viventem/MetaRadar#152]
Feature/Issue
*************
In the device tab, Filters don't work/work properly on the devices around me.

Preferred Outcome
******************
I'm unsure if it's intended to work this way, but it would be nice to prevent my own devices from appearing in this list. I'm unsure why the devices around me tab is treated separately from the rest of the device list

![Screenshot_20240809_215728_BLE Radar.jpg](https://github.com/user-attachments/assets/57cdf1a3-bb70-4f78-8308-a76c551fc072)

![Screenshot_20240809_215706_BLE Radar.jpg](https://github.com/user-attachments/assets/bb257d41-27b4-48c2-a1c8-630befa9671d)

**Comment by Semper-Viventem** [comment]
Good point. The filter for online devices is supported since `0.27.1-beta`
